mediatranslation/test/quickstart.test.js

```diff
@@ -23,8 +23,9 @@ const cmd = 'node quickstart.js';
 const filePath = path.join(__dirname, '..', 'resources/audio.raw');
 const exec = cmd => execSync(cmd, {encoding: 'utf-8'});
 
+// skip this test because testing code sample is deprecated
 describe('Quickstart', () => {
-  it('should translate from a streamed file', async () => {
+  xit('should translate from a streamed file', async () => {
```

Comment:
```suggestion
it.skip('should translate from a streamed file', async () => {
```

Reply:
If we're no longer running these tests due to future deprecation, it would be better to remove them fully.